vllm/v1/engine/async_llm.py

```diff
@@ -240,8 +243,24 @@ async def _generate(
         # request if we end up here.
         except asyncio.CancelledError:
             await self.abort(request_id)
+            if self.log_requests:
+                logger.info("Request %s aborted.", request_id)
             raise
 
+        # Engine is dead. Do not abort since we shut down.
+        except EngineDeadError:
+            if self.log_requests:
+                logger.info("Request %s failed.", request_id)
+            raise
+
+        # Error in the generate() task (possibly recoverable).
+        except Exception as e:
+            await self.abort(request_id)
+            if self.log_requests:
+                logger.info("Request %s failed.", request_id)
+            logger.exception("GOT EXCEPTION:", exc_info=e)
```

Comment:
Is this line meant to be here? We should probably change the message if so, and there's no need to set `exc_info=e` since it will use that by default in the except block.